streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredVersionedKeyValueStore.java

```diff
@@ -84,22 +86,37 @@
         extends MeteredKeyValueStore<K, ValueAndTimestamp<V>> {
 
         private final VersionedBytesStore inner;
+        private final Serde<V> rawValueSerde;
```

Comment:
nit: why `raw` -- we usually use `raw` when we have `byte[]` at hand

Reply:
`valueSerde` is already defined in the base class, and I couldn't come up with a better name 😆 Suggestions welcome!